src/node.cc

```diff
@@ -0,0 +1,40 @@
+#include "node.h"
+#include <Eigen/Dense>
+#include <cmath>
+
+Eigen::VectorXd pipenetwork::Node::compute_poly_coefficients(
```

Comment:
Can you pass three arrays instead? Could you write the formula, so I can see what you are doing here?

Reply:
I simply did what wntr does for this part. I guess they just try to use polynomial to approximate the transition stage for of the piecewise function such as the slope behavior of (x)^-(0.5) as x approaches 0. I will write the formula though

Reply:
OK